### ISSUE TITLE
How to display validation loss/test loss along with train loss

### ISSUE THREAD

**Issue by QuickLearner171998** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#291]
I have generated tfRecords for train and test. Now when I run model_main.py it displays only the train loss. Is there any way to also display the validation loss or the test loss so that I can check it is decreasing and finally becoming comparable to the train loss.
